File: go/Begin/task_08/main.go
package main

import (
	"fmt"
)

func main() {
	var a, b float64

	fmt.Print("Enter the first number: ")
	fmt.Scan(&a)
	fmt.Print("Enter the second number: ")
	fmt.Scan(&b)

	avg := (a + b) / 2
	fmt.Printf("Avarage value: %.2f\n", avg)
}

File: java/Begin/task_08/main.java
import java.util.Scanner;

public  class main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter the first number: ");
        double num1 = scanner.nextDouble();

        System.out.print("Enter the second number: ");
        double num2 = scanner.nextDouble();

        double avg = (num1 + num2) / 2;
        System.out.printf("Average value: %.2f\n", avg);
    }
}